src/spectangular.js

```diff
@@ -57,6 +70,43 @@ class Spectangular {
     }, 50000, 'Was not able to load the application!');
   }
 
+  /**
+   * Refreshes the page and loads again the page. This is used in combination with closePage. This method reloads
+   * the Angular context before continue with the loading the page. This method is useful if tests use the same fixture
+   * and the test need a fully reload of the fixture and Angular context.
+   *
+   * @param url
+   * @param waitForElementSelector
+   */
+  loadPageRefresh(url, waitForElementSelector = 'body') {
+    browser.driver.navigate().refresh();
+    this.loadPage(url, waitForElementSelector);
+  }
+
+  /**
+   * Page loading of angular pages. This method is called before each tests in the beforeEach method.
+   *
+   * The page loads the url. Protractor waits until the page is loaded and expect that the web element is loaded.
+   * This method is used for application which do NOT use AMD module pattern and load the Angular application with
+   * a ng-app directive.
+   *
+   * Example:
+   *  Spectangular.loadAngularPage('http://www.my-angular-app.nl','.toolbar');
+   *
+   * @param url of the page
+   * @param waitForElementSelector, css selector for the element
+   */
+  loadAngularPage(url, waitForElementSelector = 'body') {
+    browser.get(`${this.baseUrl}${url}`);
+    expect(element(by.css(waitForElementSelector)).isPresent()).toBe(true);
+  }
```

Comment:
What does this function do different then loadPage? Both do browser.get and then wait for an element to be present...